eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -135,7 +135,13 @@ jobs:
 
       - pwsh: |
           $(Build.SourcesDirectory)/eng/tools/compress-coverage.ps1 "coverage" "$(Build.ArtifactStagingDirectory)/CodeCoverage"
-        displayName: "Copy Coverage"
+        displayName: "Copy Coverage Reports - Node"
+        condition: and(always(), eq(variables['TestType'], 'node'))
```

Comment:
Why `always()` as opposed to `succeeded()`?

Reply:
I think I took that from another language's pipeline I refered to for coverage reports

Reply:
this one is outdated one though